tests/step-108.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+# Test:
+#   Custom installation prefix test
+
+INSTALL_DIR="/tmp/githooks"
+sh /var/lib/githooks/install.sh --prefix "$INSTALL_DIR" || exit 1
+
+if [ ! -d "$INSTALL_DIR/.githooks" ] || [ ! -f "$INSTALL_DIR/.githooks/bin/githooks" ]; then
+    echo "! Expected the install directory to be in \`$INSTALL_DIR\`"
+    exit 1
+fi
+
+if [ "$(git config --global githooks.installDir)" != "$INSTALL_DIR/.githooks" ]; then
+    echo "! Install directory in config \`$(git config --global githooks.installDir)\` "
+    echo "  is incorrect!"
```

Comment:
Keep this on the previous line, the error printed will be the last line, so better not to break it into two.